ci-operator/jobs/openshift/origin/openshift-origin-main-presubmits.yaml

```diff
@@ -3779,6 +3779,82 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-metal-ipi-ovn-dualstack,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
`always_run: false` on all?

Reply:
++ always_run false

Reply:
I suppose it's the default value. I should explicitly set `always_run: false` in the job config.